Marsiple/App/Post Details/PostDetailsViewController.swift

```diff
@@ -29,6 +29,10 @@ class PostDetailsViewController: UIViewController {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
Just noticed this. We will fire off two requests when first showing the `PostDetailsViewController` - via the `initialiser` and `viewWillAppear`. 

Since you're using `viewWillAppear` to initiate a `reload` once a comment was posted, we should remove this and use a `callback via a closure`, from the `CommentViewController` to the `PostDetailsViewController`, to signal that the content should be refetched and reloaded.